### PR TITLE
Update index.html

### PR DESCRIPTION
**Pull request by AprilGitHubTest**
I just compared and pulled the 2 branches.

### CODE REVIEW

**Comment by AlexGitHubTest** [comment]
I accepted your pull request, they are now merged.

**Review by AlexGitHubTest — COMMENTED** [review]
All looks good!

File: index.html
Comment (AprilGitHubTest) [inline index.html:1]: Could you perhaps get rid of the red-line?

**Review by AprilGitHubTest — COMMENTED** [review]
Whenever you resolve the issue, press the resolve conversation button.

File: index.html
Comment (AlexGitHubTest) [reply index.html:1]: Of course, I'll get rid of it in just a moment!

**Pull request merged**